acceptance/tests/i18n/disable_option_disable_i18n.rb

```diff
@@ -0,0 +1,44 @@
+test_name 'C100562: verify that disable_i18n can be set to false and have translation still enabled' do
+  confine :except, :platform => /^eos-/ # translation not supported
+  confine :except, :platform => /^cisco_/ # translation not supported
+  confine :except, :platform => /^cumulus/ # translation not supported
+  confine :except, :platform => /^solaris/ # translation not supported
+  confine :except, :platform => /^aix/ # QENG-5283 needed for this to work
```

Comment:
Do you need these confines, given the `skip_test` line down below? (Same note about that utility function possibly not returning nil in all the missing locale cases)

Reply:
By confining the tests out they'll run faster on those machines as we won't have to run commands on the nodes to figure out its not supported, and except for AIX I don't think likely to be supported any time soon.